### ISSUE TITLE
Is there a pytorch version

### ISSUE THREAD

**Issue by Songxinlei** [mlyg/unified-focal-loss#2]
Is there a pytorch version？

**Comment by mlyg** [comment]
I am afraid not at the moment. However, I will be working on translating these loss functions into pytorch soon so I will update the repository once that is ready. 

In a previous issue opened by another user, it appears they have converted the 'Focal Tversky loss' into pytorch - feel free to take a look and see whether it is something you can use in the meantime.
